### PR TITLE
[Refactor:Submission] Change regrade to grade_inquiry

### PR DESCRIPTION
**Pull request by NiaHeermance**
### What is the current behavior?
While the Grade Inquiry feature has been by its current name to users for a while, in the code, it was referred to by "regrade_request" or sometimes simply "regrade." This name was confusing because the term "regrade" is already used for adding submissions to the grading queue.

### What is the new behavior?
All appropriate places in the code and database have been renamed from "regrade_request" to "grade_inquiry".

### CODE REVIEW

**Comment by musaabimran** [comment]
I've checked out the PR, and it's spot-on! It was totally necessary because that word "regrade" was causing a ton of issues. The changes made in the PR tackle those problems.

**Review by musaabimran — APPROVED** [review]
I've checked out the PR, and it's spot-on! It was totally necessary because that word "regrade" was causing a ton of issues. The changes made in the PR tackle those problems.

**Review by mahi-pas — CHANGES_REQUESTED** [review]

**Review by mahi-pas — CHANGES_REQUESTED** [review]

File: site/public/js/server.js
[the remark this answers is not in the document]
Comment (NiaHeermance) [reply site/public/js/server.js:1]: Actually, that function is no longer used, so I have deleted it and pushed a new commit. Thanks for the catch!

**Comment by mahi-pas** [comment]
Did some basic grade inquiry testing. Looks good to me

**Review by mahi-pas — APPROVED** [review]
LGTM

**Pull request merged**